src/test/java/uk/gov/pay/directdebit/payments/dao/PaymentViewDaoIT.java

```diff
@@ -434,32 +402,23 @@ public void shouldReturn6Records_whenSearchingByFailedExternalState() {
                 .withMandateFixture(mandateFixture)
                 .withState(PaymentState.CREATED)
                 .insert(testContext.getJdbi());
-        for (int i = 0; i < 6; i++) {
-            if (i % 3 == 0) {
-                aPaymentFixture()
-                        .withMandateFixture(mandateFixture)
-                        .withState(PaymentState.CANCELLED)
-                        .insert(testContext.getJdbi());
-                continue;
-            }
-            if (i % 2 == 0) {
-                aPaymentFixture()
-                        .withMandateFixture(mandateFixture)
-                        .withState(PaymentState.EXPIRED)
-                        .insert(testContext.getJdbi());
-                continue;
-            }
+        
+        List.of(1,2).forEach(i -> {
+            aPaymentFixture()
+                    .withMandateFixture(mandateFixture)
+                    .withState(PaymentState.CANCELLED)
+                    .insert(testContext.getJdbi());
             aPaymentFixture()
                     .withMandateFixture(mandateFixture)
                     .withState(PaymentState.FAILED)
                     .insert(testContext.getJdbi());
-        }
-
+        });
+        
```

Comment:
bit of extra white space :)

Reply:
i'll rectify this in the next PR as I don't think it's worth triggering another build for this :)